src/components/atoms/Icon/icons.ts

```diff
@@ -198,6 +198,8 @@ export default {
   export: Export,
   infobox: Infobox,
   text: InfoText,
+  // FIXME(@keiya01): Use HTML icon
+  // html: InfoText,
```

Comment:
I will fix this before merging

Reply:
[feat: add html icon](https://github.com/reearth/reearth-web/pull/454/commits/ad5b99e8af89a71b9a3d99c40e0b56b80395deb8)